docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -114,6 +114,18 @@ Manual guides for deployment on various platforms, for One-click and docker plea
 	</InstallLink>
 </div>
 
+<div>
+	<InstallLink link="../deployment/qovery.html">
```

Comment:
This link is wrong it should be
`deployment/hosting-guides/qovery.html`

Reply:
ahah good catch